frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -87,6 +100,99 @@ class OverrideForkPointDialog(
             thisRowComment
           )
       }
+
+      row {
+        radioButton(
+          format(
+            getString(
+              "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom"
+            ),
+            radioButtonComment
+          ),
+          OverrideOption.CUSTOM
+        )
+          .comment(
+            thisRowComment
+          )
+
+        comboBox<ICommitOfManagedBranch?>(
+          (listOf(branch.forkPoint, parentBranch.pointedCommit) + branch.commits.toMutableList()).filterNotNull(),
+          object : DefaultListCellRenderer() {
+            private val defaultBackground = UIManager.get("List.background") as Color
+            override fun getListCellRendererComponent(
+              list: JList<*>?,
+              value: Any?,
+              index: Int,
+              isSelected: Boolean,
+              cellHasFocus: Boolean
+            ): Component {
+              val commit: ICommitOfManagedBranch? = value as ICommitOfManagedBranch?
+              componentOrientation = list!!.componentOrientation
+
+              var bg: Color? = null
+              var fg: Color? = null
+
+              val dropLocation = list.dropLocation
+              var customIsSelected = isSelected
+              if (dropLocation != null && !dropLocation.isInsert && dropLocation.index == index) {
+                bg = list.selectionBackground
+                fg = list.selectionForeground
+                customIsSelected = true
+              }
+
+              if (customIsSelected) {
+                background = bg ?: list.selectionBackground
+                foreground = fg ?: list.selectionForeground
+                font = Font(list.font.name, Font.BOLD, list.font.size)
+              } else {
+                font = list.font
+                background = list.background
+                foreground = list.foreground
+              }
+
+              icon = null
+
+              isEnabled = list.isEnabled
+
+              text = if (commit != null) {
+                var prefix =
```

Comment:
Hmmm currently the combo box list looks like that:

![image](https://user-images.githubusercontent.com/3383210/194767893-1ba6164e-67e7-4121-867d-64997dd0fea5.png)

for list of commits like

![image](https://user-images.githubusercontent.com/3383210/194767911-d0a1f502-dfda-4b52-8cdc-5a176d65c158.png)

Reply:
I'd rather avoid including _Inferred commit_ at the beginning of the list by default (?)

Also, let's order the commits in the same order as they appear in the branch status: from oldest ("closest" to the parent branch) to the newest ("closest" to the child branch).

Reply:
yup, lets keep the order